services/user.py

```diff
@@ -0,0 +1,52 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = "",
+        last_name: str = ""
+) -> User:
+    return get_user_model().objects.create_user(
+        username=username,
+        password=password,
+        email=email,
+        first_name=first_name,
+        last_name=last_name
+    )
+
+
+def get_user(user_id: int) -> User:
+    try:
+        return get_user_model().objects.get(pk=user_id)
+    except User.DoesNotExist:
+        print(f"User with id {user_id} does not exist")
```

Comment:
Instead of printing an error message when a user does not exist, consider raising an exception or returning a more informative response to handle this scenario more gracefully.